lib/nexmo_developer/app/services/translator/smartling_downloader.rb

```diff
@@ -0,0 +1,55 @@
+module Translator
+  class SmartlingDownloader
+    include ::Translator::Utils
+
+    attr_reader :file_uris
+
+    def self.call(attrs = {})
+      new(attrs).call
+    end
+
+    def initialize(file_uris: nil)
```

Comment:
@bencgreenberg `file_paths` was renamed to `file_uris_from_smartling` can you update this? And let's add a spec for this scenario as well

Reply:
Good catch. What would the spec cover specifically since this was an omission during a one-time renaming?

Reply:
initializing the object without a parameter, we can check that the API is called

Reply:
👍

Reply:
Added the spec, @fabianrbz